### PR TITLE
Update classes on protobufs update

### PR DESCRIPTION
**Pull request by scruplelesswizard**
This PR changes the update process for Protobufs. Every time a change is made to the `meshtastic` submodule in a PR, the `regen-protos.sh` will be run and the changes committed to the PR's branch.

This enables class regeneration on Dependabot's PRs and also on any other users PR where `meshtastic/` is updated. Additionally it ensures that on any PRs where the protobufs have changed that the classes match.

### CODE REVIEW

**Review by thebentern — APPROVED** [review]

File: .github/workflows/update_protobufs.yml [hunk 1]
```diff
@@ -25,10 +26,11 @@ jobs:
...
         with:
-          title: Update protobufs and classes
-          add-paths: |
-            protobufs
-            src/mesh
+          add: src/mesh
+          author_name: CI Bot
+          author_email: meshtastic-ci-bot@users.noreply.github.com
+          default_author: github_actor
+          message: Update classes from protobugs
```
Comment (scruplelesswizard) [inline .github/workflows/update_protobufs.yml:36]:
```suggestion
          message: Update classes from protobufs
```

**Comment by caveman99** [comment]
IS this correct? the complete submodule path is /protobufs/meshtastic/**

**Comment by scruplelesswizard** [comment]
Oh, totally not. I'll get a patch up shortly! Thanks for checking @caveman99

**Comment by scruplelesswizard** [comment]
PR closed unintentionally

**Review by caveman99 — APPROVED** [review]

**Pull request merged**